FirebaseAdmin/FirebaseAdmin.IntegrationTests/FirebaseAuthTest.cs

```diff
@@ -510,6 +510,115 @@ public async Task ListUsers()
             }
         }
 
+        [Fact]
+        public async Task ImportUsersBasic()
+        {
+            var randomUser = RandomUser.Create();
+            var args = new ImportUserRecordArgs()
+            {
+                Uid = randomUser.Uid,
+                Email = randomUser.Email,
+                DisplayName = "Random User",
+                PhotoUrl = "https://example.com/photo.png",
+                EmailVerified = true,
+            };
+
+            var options = new UserImportOptions();
+            IEnumerable<ImportUserRecordArgs> usersLst = new List<ImportUserRecordArgs>();
+            usersLst = usersLst.Append(args);
+
+            var resp = await FirebaseAuth.DefaultInstance.ImportUsersAsync(usersLst);
+
+            try
+            {
+                Assert.Equal(1, resp.SuccessCount);
+                Assert.Equal(0, resp.FailureCount);
+            }
+            finally
+            {
+                await FirebaseAuth.DefaultInstance.DeleteUserAsync(randomUser.Uid);
+            }
+        }
+
+        [Fact]
+        public async Task ImportUsersPasswordNoHash()
+        {
+            var randomUser = RandomUser.Create();
+            var args = new ImportUserRecordArgs()
+            {
+                Uid = randomUser.Uid,
+                Email = randomUser.Email,
+                PasswordSalt = Encoding.ASCII.GetBytes("abc"),
+                PasswordHash = Encoding.ASCII.GetBytes("def"),
+            };
+
+            var options = new UserImportOptions();
+            IEnumerable<ImportUserRecordArgs> usersLst = new List<ImportUserRecordArgs>();
+            usersLst = usersLst.Append(args);
+            await Assert.ThrowsAsync<ArgumentNullException>(
+                async () => await FirebaseAuth.DefaultInstance.ImportUsersAsync(usersLst, options));
+        }
+
+        [Fact]
+        public async Task ImportUsersPassword()
+        {
+            var randomUser = RandomUser.Create();
+            var args = new ImportUserRecordArgs()
+            {
+                Uid = randomUser.Uid,
+                Email = randomUser.Email,
+                DisplayName = "Random User",
+                PhotoUrl = "https://example.com/photo.png",
+                EmailVerified = true,
+                PasswordSalt = Encoding.ASCII.GetBytes("abc"),
+                PasswordHash = Encoding.ASCII.GetBytes("def"),
+                CustomClaims = new Dictionary<string, object>()
+                {
+                    { "admin", true },
+                },
+                UserProviders = new List<UserProvider>
+                {
+                    new UserProvider()
+                    {
+                        Uid = "google.uid",
```

Comment:
Use `randomUser.Uid` and `randomUser.Email` to be on the safe side.

Reply:
Should I use `randomUser.Uid` and `randomUser.Email` for the second `UserProvider` as well? Or is it ok to keep it as is? I'm not sure what the difference is

Reply:
Yeah use random values for all.